apps/condo/domains/meter/utils/testSchema/index.js

```diff
@@ -78,6 +81,23 @@ async function updateTestMeterReadingSource (client, id, extraAttrs = {}) {
     return [obj, attrs]
 }
 
+async function createTestMeterWithResident (extraMeterAttrs = {}) {
```

Comment:
If you don't pass the `client` as param you need to use `make` prefix. Like `makeClientWithResidentAndMeter()`

Reply:
Done